test/containerfiles/bundle-passes copy.Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM scratch
+
+# Core bundle labels.
+LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
+LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
+LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
+LABEL operators.operatorframework.io.bundle.package.v1=sample-operator
+LABEL operators.operatorframework.io.bundle.channels.v1=alpha
+LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.7.1+git
+LABEL operators.operatorframework.io.metrics.mediatype.v1=metrics+v1
+LABEL operators.operatorframework.io.metrics.project_layout=go.kubebuilder.io/v3
+
+# Labels for testing.
+LABEL operators.operatorframework.io.test.mediatype.v1=scorecard+v1
+LABEL operators.operatorframework.io.test.config.v1=tests/scorecard/
+
+# Copy files to locations specified by labels.
+COPY successful-bundle-assets/manifests /manifests/
+COPY successful-bundle-assets/metadata /metadata/
+COPY successful-bundle-assets/tests/scorecard /tests/scorecard/
```

Comment:
I don't see a reference to /licenses?

Reply:
AFAIK, Licenses are required for container checks and not for bundles. The container-passes.Dockerfile puts this in place.

Reply:
Cool. WFM.